interview-questions.md

```diff
@@ -16,42 +16,74 @@ Researched answer:
 
 Your answer:
 
+Git is an extension that gives the develper to test code on a console or terminal.
+
 Researched answer:
 
+Git is a version control system used for tracking changes in computer files.
+
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: if/else statement, equal to, less than, greater than, greater than or equal to, and less than or equal to.
 
-Researched answer:
+Researched answer: logical operators
 
-1. What is the difference between an array's index and its value?
+3. What is the difference between an array's index and its value?
 
-Your answer:
+Your answer: Index is the count of data in the brackets[] from 0 to the amount of data in the array. The value is what is displayed by the data and what the equal operator has it equal to.
 
 Researched answer:
 
+An array index is an integer indicating a position in an array. The value is the number of elements in an array.
+
 4. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
 
 Your answer:
 
+Comparison:
+
+Arrays and Strings have an index and have .length().
+
 Researched answer:
 
+they both are a collection of items, both have a length property, both are zer-indexed.
+
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-Your answer:
+Your answer: Proper techniques are communication, respect, and teamwork to execute the project. One driver and one navigator.
 
 Researched answer:
 
+Pair programming is a software development technique in which two programmers work together at one workstation. One, the driver, writes code while the other, the observer or navigator.
```

Comment:
Not observer, the navigator is responsible for many different things while pair programming, they are not just there to observe!